src/server/routes/collection.js

```diff
@@ -55,6 +79,42 @@ router.param(
 	middleware.makeCollectionLoader()
 );
 
+router.get('/:collectionId', auth.isAuthenticatedForCollectionView, async (req, res) => {
+	const {collection} = res.locals;
+	if (collection.entityType !== 'Edition' && collection.entityType !== 'Work') {
+		return res.status(200).send(collection);
+	}
+	const {orm} = req.app.locals;
+	const relations = getEntityRelations(collection.entityType);
+
+	const entitiesPromise = collection.items.map(item => orm.func.entity.getEntity(orm, collection.entityType, item.bbid, relations));
```

Comment:
In the future we might want to think about optimizing this to fetch all the entities in a single batch, but that's for another day.

Reply:
I'll add this to that doc